File: ts/iq/w-comms-chat~get~blocking_status+query.ts
/** <iq type="get" xmlns="w:comms:chat">. Replies with one of: Success, ServerError. */
export function buildPsaChatBlockGet(): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'w:comms:chat', type: 'get' }, content: [
      { tag: 'query', attrs: {  } },
    ] }
}

File: rust/iq/w-comms-chat~get~blocking_status+query.rs
/// <iq type="get" xmlns="w:comms:chat">. Replies with one of: Success, ServerError.
pub fn build_psa_chat_block_get() -> Node {
    NodeBuilder::new("iq")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "w:comms:chat")
        .attr("type", "get")
        .child(NodeBuilder::new("query").build())
        .build()
}